genai/api/npc_chat_api/k8s.yaml

```diff
@@ -1,3 +1,159 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: npc-config
+data:
+  config.toml: |
+    # Copyright 2024 Google LLC All Rights Reserved.
```

Comment:
I would just move the copyright to this file (and below) and delete from the ConfigMaps. Otherwise LGTM